src/main/java/org/craftercms/core/processors/impl/SkipAllItemProcessor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2007-2023 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.craftercms.core.processors.impl;
+
+import org.craftercms.core.exception.ItemProcessingException;
+import org.craftercms.core.processors.ItemProcessor;
+import org.craftercms.core.service.CachingOptions;
+import org.craftercms.core.service.Context;
+import org.craftercms.core.service.Item;
+
+/**
+ * No-op {@link ItemProcessor} exclusive implementation.
```

Comment:
I think it would be helpful to explain this processor and why you might use it a little further.

Looking at an indication doesn't do much to self document what it's role is:

contentStoreService.getItem(context, DEFAULT_CACHING_OPTIONS, CONTENT_DESCRIPTOR_PATH, new SkipAllItemProcessor(), true);


So I am asking to get an item and I am passing an exclusive "SkipAllItemProcessor" ... Just reading the code with "fresh eyes" I have to say... huh?